aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnActionRegistry.js

```diff
@@ -38,12 +39,41 @@
             return action;
         }
 
+        /**
+         * Function to remove symbols that don't match the pattern
+         * @param {string} target - target string
+         * @param {RegExp} pattern -  pattern for correct symbols
+         * */
+        static _replaceForbiddenSymbols(target, pattern) {
```

Comment:
Consider renaming to "sanitizeName"
And getting rid of pattern param as it does not change (use constant instead)